main.go

```diff
@@ -147,7 +147,8 @@ func main() {
 
 	// Mount middleware
 	service.Use(middleware.RequestID())
-	service.Use(middleware.LogRequest(configuration.IsPostgresDeveloperModeEnabled()))
+	// Use our own log request to inject identity id and modify other properties
+	service.Use(log.LogRequest(configuration.IsPostgresDeveloperModeEnabled()))
```

Comment:
@kwk maybe we can see if in the future we need to have a separate env to enable/disable this log. Let's keep it as it was atm.